src/contracts/LedaNFT-address.json

```diff
@@ -1,3 +1,3 @@
 {
-  "address": "0xe8b98E987DAD25981dB8d3fd37cEBf16087dB4CF"
+  "address": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0"
```

Comment:
Be careful if you send this to main. The front end in production needs to point to the address in the Goerli test network.